ariadne/wsgi.py

```diff
@@ -16,9 +16,6 @@
     CONTENT_TYPE_TEXT_PLAIN,
     DATA_TYPE_JSON,
     DATA_TYPE_MULTIPART,
-    HTTP_STATUS_200_OK,
-    HTTP_STATUS_400_BAD_REQUEST,
-    HTTP_STATUS_405_METHOD_NOT_ALLOWED,
 )
 from .exceptions import HttpBadRequestError, HttpError
```

Comment:
We should have a blank line here to separate 3rd party and 1st party imports. I don't know why its missing.